RubberduckBaseProject.csproj

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <Company>Rubberduck-VBA</Company>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <TargetFramework>net46</TargetFramework>
+    <LangVersion>7.2</LangVersion>
+    <SolutionDir>$(MSBuildProjectDirectory)</SolutionDir>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <OutputPath>bin\$(Configuration)\</OutputPath>
+    <RunCodeAnalysis>true</RunCodeAnalysis>
+    <RestorePackages>true</RestorePackages>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(AssemblyVersion)' == ''">
+    <AssemblyVersion>2.2.*</AssemblyVersion>
```

Comment:
FWIW, this causes VS to emit a warning for each project about it being not a typical format.

Reply:
Added a comment about that. There's also something to note about how appveyor changes the version of the assemblies. It patches the AssemblyInfo.cs file of Rubberduck.Main. 
Unfortunately that no longer is a suitable option. I'll need to peruse AV docs on how to fix that.

Reply:
Is this resolved? How do we make RD v2.3.0?